tools/qfvis/source/flow.c

```diff
@@ -336,6 +564,18 @@ RecursiveClusterFlow (int clusternum, threaddata_t *thread, pstack_t *prevstack)
 		thread->base->numcansee++;
 	}
 
+	// check all target portals instead of just neighbor portals, if the time is right
+	if (options.targetratio > 0.0 &&
+		prevstack->num_expected_targetchecks > 0 &&
+		thread->numsteps * options.targetratio >=
+		thread->numtargetchecks + prevstack->num_expected_targetchecks)
+	{
+		unsigned num_actual_targetchecks = IterativeTargetChecks(thread, thread->pstack_head.next);
+		// thread->stats.c_targetcheck += num_actual_targetchecks;
```

Comment:
With the exception of debug code, unused code should be deleted.

Reply:
I've ignored all stat counters for the first version (except for the semantically relevant `return thread->stats.portalcheck - initial_portalcheck;`) Do you have opinions on which things you'd like to be measured?

Reply:
I think it would be good to count anything culled this way. Otherwise, I don't know.